### PR TITLE
Added naming to migration bundles

### PR DESCRIPTION
**Pull request by souravmitra**
In similar lines to HibernateBundles, having the possibility of unique names. This PR attempts to do the same for MigrationBundles, so that multiple migration bundles can be provided for.

### CODE REVIEW

File: dropwizard-migrations/src/main/java/io/dropwizard/migrations/MigrationsBundle.java [hunk 1]
```diff
@@ -7,11 +7,17 @@
...
+
     @Override
     @SuppressWarnings("unchecked")
     public final void initialize(Bootstrap<?> bootstrap) {
         final Class<T> klass = (Class<T>) bootstrap.getApplication().getConfigurationClass();
-        bootstrap.addCommand(new DbCommand<>(this, klass));
+        bootstrap.addCommand(new DbCommand<>(name(), this, klass));
+    }
+
+    protected String name() {
```
Comment (jplock) [inline dropwizard-migrations/src/main/java/io/dropwizard/migrations/MigrationsBundle.java]: can you make this `public`?

File: dropwizard-migrations/src/main/java/io/dropwizard/migrations/MigrationsBundle.java [hunk 2]
```diff
@@ -7,11 +7,17 @@
 import io.dropwizard.setup.Environment;
 
 public abstract class MigrationsBundle<T extends Configuration> implements Bundle, DatabaseConfiguration<T> {
+    public static final String DEFAULT_NAME = "db";
```
Comment (jplock) [inline dropwizard-migrations/src/main/java/io/dropwizard/migrations/MigrationsBundle.java]: can you make this `private`?

**Comment by jplock** [comment]
I think this is a good improvement.  Thanks for the contribution!  Just left some minor comments.

File: dropwizard-migrations/src/main/java/io/dropwizard/migrations/MigrationsBundle.java (hunk 1 above)
Comment (souravmitra) [inline dropwizard-migrations/src/main/java/io/dropwizard/migrations/MigrationsBundle.java]: @jplock : Thanks for the comments. Do you see usage of name() outside subclasses of MigrationBundle?

Comment (jplock) [inline dropwizard-migrations/src/main/java/io/dropwizard/migrations/MigrationsBundle.java]: No, but to stay consistent with `initialize()` and `run()` and how its defined in https://github.com/dropwizard/dropwizard/blob/master/dropwizard-core/src/main/java/io/dropwizard/Application.java

Comment (souravmitra) [inline dropwizard-migrations/src/main/java/io/dropwizard/migrations/MigrationsBundle.java]: Made the change. Thanks

File: dropwizard-migrations/src/main/java/io/dropwizard/migrations/MigrationsBundle.java (hunk 2 above)
Comment (souravmitra) [inline dropwizard-migrations/src/main/java/io/dropwizard/migrations/MigrationsBundle.java]: Done

**Comment by jplock** [comment]
Sorry, one more minor change, would you mind updating https://github.com/dropwizard/dropwizard/blob/master/docs/source/about/release-notes.rst with a note about this new functionality and reference to this PR?

**Comment by souravmitra** [comment]
@jplock Added to the release notes. Thanks.

**Comment by jplock** [comment]
Looks great, thanks for contributing!

**Pull request merged**